src/gui/addnewtorrentdialog.cpp

```diff
@@ -66,6 +67,7 @@ const QString KEY_WIDTH = SETTINGS_KEY("Width");
 const QString KEY_EXPANDED = SETTINGS_KEY("Expanded");
 const QString KEY_TOPLEVEL = SETTINGS_KEY("TopLevel");
 const QString KEY_SAVEPATHHISTORY = SETTINGS_KEY("SavePathHistory");
+const char *KEY_SAVEPATHHISTORYLENGTH = SETTINGS_KEY("SavePathHistoryLength");
```

Comment:
I would use `KEY_SAVEPATHHISTORYLENGTH[]`

Reply:
Hey, guys, let's see avove! We use QString for all other similar constants. Why to use another type here?

Reply:
@Chocobo1: `const char[]` would create an additional variable AFAIK, `const char*` just points to the location of the literal.
@glassez: `CachedSettingValue` needs `const char*`.

Reply:
with `const char*`, the pointer can be changed to point elsewhere and this won't be the case if using array... unless using `const char * const` and thats a mouthful...

Reply:
Let it be const char[] in anonymous namespace then.